game/dino_jump_main.py

```diff
@@ -0,0 +1,394 @@
+#############################################################################
+# IMPORTS
+#############################################################################
+
+import sys
+import random
+import numpy as np
+import pygame
+
+from sprites import import_sprite
+
+
+#############################################################################
+# GAME MODULE
+#############################################################################
+
+
+class Game:
+
+    def __init__(self):
+
+        # Editable Constants
+        self.GRID_SIZE = 4  # scale entire pygame panel size (only for online display)
+        self.WIDTH, self.HEIGHT = (
+            150,
+            48,
+        )  # define pixel height and width for actual board
+        self.GROUND_Y, self.LOW_BIRD_Y, self.HIGH_BIRD_Y = (
+            2,
+            18,
+            28,
+        )  # proportions for the three axes of obstacles
+
+        # Non-Editable Constants
+        self.BG_COLOR = (255, 255, 255)  # white colour (background)
+        self.BLOCK_COLOR = (0, 0, 0)  # black colour (sprites)
+        self.FPS = 60  # frames per second frames update
+        self.GROUND_AXIS = (
+            self.HEIGHT - self.GROUND_Y
+        )  # defines axis height with (0,0) lower left corner
+        self.LOW_BIRD_AXIS = (
+            self.HEIGHT - self.LOW_BIRD_Y
+        )  # defines axis height with (0,0) lower left corner
+        self.HIGH_BIRD_AXIS = (
+            self.HEIGHT - self.HIGH_BIRD_Y
+        )  # defines axis height with (0,0) lower left corner

```

Comment:
This is my mistake. I forgot that (0, 0) is the top left corner. I think to be consistent with the rest of the code base meaning of "y-coordinates", `self.GROUND_Y, self.LOW_BIRD_Y, self.HIGH_BIRD_Y` should be called something else?(maybe like `self.GROUND_RATIO` or `self.GROUND_PROPORTION`) 
and `self.GROUND_AXIS, self.LOW_BIRD_AXIS, self.HIGH_BIRD_AXIS` should be the ones called `self.GROUND_Y, self.LOW_BIRD_Y, self.HIGH_BIRD_Y`, since they are the actual values used to calculate y-coordinates of objects?

Reply:
All of these changes are made!